### PR TITLE
Create an entry for Ian Cooper on the speaker list

### PR DESCRIPTION
**Pull request by iancooper**

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=534) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=534) before we can accept your contribution.<br/><br/>:x: iancooper [sign now](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=534)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-foundation/website?pullRequest=534) it.</sub>

**Comment by iancooper** [comment]
Can't really tell if there is something in the file it does not like. I did use markdown heading syntax...

**Comment by daveaglick** [comment]
You broke Statiq! I'll dig into the underlying issue with reporting validation errors - what _should_ have happened is a nice verification check message telling you the Twitter link needs to be a full URL, I.e. `https://twitter.com/ICooper`.

**Comment by daveaglick** [comment]
Ah, I actually see there's a different error reported above:

```
[ERRO] Inputs/Process » ExecuteIf » ExtractFrontMatter » ParseYaml » Document [/home/runner/work/website/website/input/community/speakers/ian-cooper.md => community/speakers/ian-cooper.md]: (Line: 16, Col: 10, Idx: 287) - (Line: 16, Col: 11, Idx: 288): While scanning for the next token, found character that cannot start any token.
[ERRO] Inputs/Process » ExecuteIf » ExtractFrontMatter » Error while executing module ParseYaml in Inputs/Process: Exception of type 'Statiq.Common.LoggedException' was thrown.
```

Looks like the YAML parser didn't like the `@` symbol for the first character of the `Twitter` value either - but that'll resolve when it's changed to a full URL as well.

In the meantime I'll look at cleaning up the error reporting in this case, it's obviously not ideal.

**Comment by iancooper** [comment]
Cool. Knowing what to fix is going to help  :-)

File: input/community/speakers/ian-cooper.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  - .NET
+  - ASP.NET
+  - ASP.NET Web API
+  - Architecture
+  - C#
+  - Containers
+  - Microservices
+  - Serverless
+Blog: https://medium.com/altdotnet/archive
+Twitter: @ICooper
```
Comment (SeanKilleen) [inline input/community/speakers/ian-cooper.md:17]:
```suggestion
Twitter: https://twitter.com/ICooper
```

File: input/community/speakers/ian-cooper.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+GitHub: https://github.com/iancooper
+LinkedIn: https://www.linkedin.com/in/ian-cooper-2b059b/?originalSubdomain=uk
+Sessionize: https://sessionize.com/ian-cooper/
+YouTube: https://www.youtube.com/playlist?list=PLZgLq02FHsbcScJaUjITEHS0Y0dmglavP
+---
+# Short Bio
+
+Polyglot Coding Architect in London, founder of #ldnug, speaker, tabletop gamer, geek. Tattooed, pierced, and bearded. The 'guv' on @BrighterCommand. He/him.
```
Comment (SeanKilleen) [inline input/community/speakers/ian-cooper.md:25]:
```suggestion
Polyglot Coding Architect in London, founder of #ldnug, speaker, tabletop gamer, geek. Tattooed, pierced, and bearded. The 'guv' on [BrighterCommand](https://twitter.com/BrighterCommand). He/him.
```

**Comment by SeanKilleen** [comment]
Added a few suggestions to try to fix the build. Sorry, didn't see the prior chat!

File: input/community/speakers/ian-cooper.md (hunk 2 above)
Comment (iancooper) [reply input/community/speakers/ian-cooper.md:25]: Thanks @SeanKilleen

**Pull request merged**